NetExercise/Array/Exercise10Array.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+
+namespace NetExercise.Array
+{
+    public class Exercise10Array
+    {
+        public static int SumSquareNumber(int[] arr)
+        {
+            int sum = 0;
+            
+            for (int i = 0; i < arr.Length; i++)
+            {
+                if (CheckSquareNumber(arr[i]))
+                {
+                    sum += arr[i];
+                }
+            }
+
+            return sum;
+        }
+        
+        public static bool CheckSquareNumber(int n)
+        {
+            var number = Convert.ToInt32(SquareRoot(Convert.ToDouble(n)));
+            return (number * number == n) ? true : false;
+        }
+        
+        public static double SquareRoot(double a)
+        {
+            var result = 1.0;
+            
+            while (Absolute(result*result - a) / a >= 0.0001)
```

Comment:
Thắc mắc, sao anh không thế này cho nhanh
```suggestion
            while (Math.Abs(result*result - a) / a >= 0.0001)
```

Reply:
vì a nghĩ mấy hàm hệ thống khi truy xuất sẽ lâu hơn,nên a viết lại để truy xuất,chứ dùng mấy hàm đó thì a biết nhưng a sợ làm ảnh hưởng tới tốc độ chương trình nên a không dùng :)

Reply:
Không có đâu, hàm hệ thống thậm chí còn chạy nhanh hơn code của mình vì nó được implement bằng ngôn ngữ C++

Reply:
ok,rứa thì khoẻ rầu :),bữa ni khỏi code lại,thanks e :)

Reply:
done